### DISCUSSION TITLE
Constraint...  profiles?  Methods? Classes?  What should we call these?

### DISCUSSION THREAD

**Discussion by DanielPuthawala** [ga4gh/cat-vrs#24] [General]
When @ahwagner and I were initially sketching out the constraint model of Cat-VRS, we built in a way to define a set of constraints that must be satisfied to include a given CatVar as being a member of some broad class of particular interest to an implementer.  Notable examples include the `Canonical Alleles` and `Categorical CNVs` for ClinVar (@larrybabb).  We named these things "profiles", alluding to the function of, eg, molecular profiles in CIViC.  

Do we like "profiles"?  Do we hate "profiles"?  Should we call them something else like "methods" or something?  I would love input here from people, as we would like to call them something that makes intuitive sense to people, but also hopefully won't be super confusing for one or another major group of people we're working with, eg, developers and implementers.

**Comment by DanielPuthawala** [comment]
@larrybabb how do you feel about calling these "recipes"?  Given that it's sort of just specifying the minimum data needed to form a given class of CatVars?

**Reply by jsstevenson** [reply]
@DanielPuthawala is the concern here that "profiles" already has a designation in VA?

**Reply by DanielPuthawala** [reply]
If that was what you and Larry were objecting to before...?

**Comment by ahwagner** [comment]
These have been implemented as "profiles" for parallelism with VA-spec in #50.

**Comment by DanielPuthawala** [comment]
Through group discussion and consensus, we have dubbed these "Recipes"
